packages/gatsby-theme-project-portal/src/components/Navbar.tsx

```diff
@@ -77,7 +77,7 @@ export const NavbarLayout: FunctionComponent<NavbarLayoutProps> = ({
               {navbarOpen ? <FaTimes /> : <FaBars />}
             </button>
             <Link
-              className="block mx-4 my-auto overflow-hidden text-nav text-black font-bold flex gap-4 items-center whitespace-nowrap"
+              className="block mx-4 my-3 overflow-hidden text-nav text-black font-bold flex gap-4 items-center whitespace-nowrap"
```

Comment:
to keep the old behavior for large screens, you could add 
```suggestion
              className="block mx-4 my-3 xl:my-auto overflow-hidden text-nav text-black font-bold flex gap-4 items-center whitespace-nowrap"
```

Reply:
You might also need to edit line 88 to keep the position of the menu below the top bar:

Currently:
![Screenshot 2023-08-01 at 07 16 52](https://github.com/thepolicylab-projectportals/project-portal-theme/assets/2803227/05977105-0982-40ef-94da-38d0a81393d2)


With 5.5-rem set on line 88
```js
 <div className={"absolute top-[5.5rem]...
```
![Screenshot 2023-08-01 at 07 17 07](https://github.com/thepolicylab-projectportals/project-portal-theme/assets/2803227/3576b012-1437-431f-bc17-ff4c72d1c234)